src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
I didn't know it break things, sorry. I see other ROS message classes are imported like that in blender_api_msgs above, so importing ChatMessage seems okay to me. Is this class actually not found on your side? I need to know how this breaks things, so I can find a solution or workaround.